.github/workflows/upgrade-test.yml

```
@@ -0,0 +1,48 @@
+# name: Upgrade Tests
```

actionlint (1.7.4)

workflow is empty

(syntax-check)